assets/src/design-system/components/notificationBubble/index.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { themeHelpers } from '../../theme';
+
+const Bubble = styled.div`
+  ${({ theme }) => css`
+    color: ${theme.colors.fg.primary};
+    background-color: ${theme.colors.accent.primary};
+    border-radius: ${theme.borders.radius.round};
+  `}
+  position: relative;
+  height: 24px;
+  width: 24px;
+`;
+
+const Inner = styled.div`
+  ${themeHelpers.fullSizeAbsolute}
+  ${themeHelpers.centerContent}
+  ${themeHelpers.expandTextPreset(
+    ({ paragraph }, sizes) => paragraph[sizes.SMALL]
+  )};
+  color: ${({ theme }) => theme.colors.bg.primary};
+`;
+
+export const NotificationBubble = ({ notificationCount }) => (
+  <Bubble>
+    <Inner>{notificationCount}</Inner>
```

Comment:
should this have any aria labeling passed in?

Reply:
Possibly? For the current use case I'm labeling the parent with the notification count so I don't think it's needed for the only use case it's currently used in. 

I'm fine with either waiting till someone needs a label on it and they can update it, or I can update this now to spread props onto the Bubble. Any preference here?

Reply:
just updated to spread props so it's more extensible. After thinking about it probably best to have things be extensible as possible if living in the ds